openbts/tests/openbts_component_tests.py

```diff
@@ -154,6 +154,38 @@ def test_get_version(self):
     self.assertTrue(self.openbts_connection.socket.recv.called)
     self.assertEqual(response.data, 'release 4.0.0.8025')
 
+class OpenBTSNominalTMSIsTestCase(unittest.TestCase):
+  """Testing the 'tmsis' command on the components.OpenBTS class."""
+
+  def setUp(self):
+    self.openbts_connection = OpenBTS()
+    # mock a zmq socket with a simple recv return value
+    self.openbts_connection.socket = mock.Mock()
+    self.openbts_connection.socket.recv.return_value = json.dumps({
+      'code': 200,
+      'data': {
```

Comment:
does it return a list of dicts or just a single dict?  (not sure, just going on the comment in the `tmsis` method above)